<?xml version="1.0"?>
<!-- Launches PyBullet simulation with MPC controller. -->

<launch>
  <arg name="config"/>
  <arg name="ctrl_config" default="default" doc="alternative controller config overwriting the one in config"/>
  <arg name="planner_config" default="default" doc="alternative planner config overwriting the one in config"/>
  <arg name="logging_sub_folder" default="default" doc="save data to a sub folder if set"/>

  <!-- Simulation -->
  <include file="$(find mm_run)/launch/pybullet_sim.launch" pass_all_args="true"/>

  <!-- Controller -->
  <include file="$(find mm_run)/launch/controller.launch" pass_all_args="true"/>

  <!-- TF -->
  <include file="$(find mm_run)/launch/thing_description.launch"/>
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="ur10_state_publisher" ns="ur10"/>
  <node name="ridgeback_state_publisher" pkg="mobile_manipulation_central" type="ridgeback_state_publisher_node.py" output="screen"/>
</launch>
